components/apps_form/apps_form_field/apps_form_field.tsx

```diff
@@ -165,21 +131,16 @@ export default class AppsFormField extends React.PureComponent<Props> {
         }
         case AppFieldTypes.CHANNEL:
         case AppFieldTypes.USER: {
-            let selectedValue: string | undefined;
-            if (this.props.value) {
-                selectedValue = (this.props.value as AppSelectOption).label;
-            }
             return (
-                <AutocompleteSelector
-                    id={name}
-                    disabled={field.readonly}
-                    providers={this.providers}
-                    onSelected={this.handleSelected}
+                <AppsFormSelectField
```

Comment:
Can we instead use `AppsFormSelectField` only one time, by combining the case statements with the static and dynamic data source ones below?